src/darwin/Framework/CHIP/templates/availability.yaml

```diff
@@ -9689,13 +9685,144 @@
       watchOS: "11.2"
       tvOS: "18.2"
   introduced:
+      clusters:
+          - MicrowaveOvenControl
+          - MicrowaveOvenMode
```

Comment:
So... now when I run codegen with this PR as of right now (rev cd2fc6b3f63aeeb051a6b99b461427609f95fc08) I see a bunch of warnings about thigns that are not annotated as either provisional, removed, or introduced, right?